### DISCUSSION TITLE
Apple M1 pro 提示“将对你的电脑造成伤害。 你应该将它移到废纸篓” 无法运行

### DISCUSSION THREAD

**Discussion by lemonlwq** [yanue/V2rayU#1245] [Q&A]
用的好好的突然打开不了 网上很多方法都试过了
sudo spctl --master-disable
sudo xattr -r -d com.apple.quarantine /Applications/V2rayU.app
也试过了 还是打不开

**Comment by ZZY1970** [comment]
ne too. M2pro, 13.4.1 (22F82)

**Reply by lemonlwq** [reply]
我装回旧版的可以用了，希望对你有用老哥（2.3.1）

**Reply by lhhxy** [reply]
你装的哪个旧版本呢？我也是这个问题

**Reply by lemonlwq** [reply]
2.3.1

**Reply by lhhxy** [reply]
谢谢，安装2.3.1可以用了

**Comment by sunformoring** [comment]
我的也是，重新装回旧版，重新签了证书还是不行
